stdlib/public/runtime/ErrorObject.mm

```diff
@@ -153,16 +167,29 @@ - (BOOL)isEqual:(id)other {
 @end
 
 Class swift::getNSErrorClass() {
-  return SWIFT_LAZY_CONSTANT([NSError class]);
+  return SWIFT_LAZY_CONSTANT(objc_lookUpClass("NSError"));
```

Comment:
Is there still a difference between this and `objc_getClass` in the modern runtime?

Reply:
Nope

Reply:
One small difference: they pass different values to the third, unused parameter of the internal function that implements the functionality. Don't ask me why.